### PR TITLE
CDAP-3173 Update upgrade document

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-3173

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -698,33 +698,20 @@ recompiled and re-deployed.)
...
-#. Run the flow queue pending metrics corrector::
-
-     $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.flow.FlowQueuePendingCorrector
-
-   This will correct the pending metrics for flows. This is a new metric that was introduced in 
-   CDAP 3.0; flows that existed before the upgrade to 3.0 do not have a correct value for this
-   metric and running the tool provides a one-time correction.
 
-#. You must recompile and then redeploy your applications. 
+#. You must recompile and then redeploy your applications that use Partition Filesets or Time Partitioned Filesets.
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: You must recompile and redeploy any applications that use either Partition Filesets or Time Partitioned Filesets.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 2]
```diff
@@ -698,33 +698,20 @@ recompiled and re-deployed.)
...
-   Please see :ref:`Configuration <install-configuration>`.
-
 #. If you are upgrading a secure Hadoop cluster, you should authenticate with ``kinit``
    before the next step (running the upgrade tool)::
 
-     $ kinit -kt <keytab> <principle>
+     $ kinit -kt <keytab> <principal>
 
-#. Run the upgrade tool::
+#. Run the upgrade tool as the user that runs CDAP master::
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: CDAP Master

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 3]
```diff
@@ -657,8 +657,8 @@ to make sure the CDAP table definitions in HBase are up-to-date.
 These steps will stop CDAP, update the installation, run an upgrade tool for the table definitions,
 and then restart CDAP.
 
-These steps will upgrade from CDAP 2.8.0 to 3.0.0. (**Note:** Apps need to be both
-recompiled and re-deployed.) 
+These steps will upgrade from CDAP 3.0.3 to 3.1.0. (**Note:** Some apps need to be both
+recompiled and re-deployed, see below.)
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: These steps will upgrade from CDAP 3.0.3 to 3.1.0. If you are on an earlier version of CDAP, please follow the upgrade instructions for the earlier versions and upgrade first to 3.0.3 before proceeding.

**Comment by awholegunch** [comment]
A few comments, then LGTM.

**Comment by poornachandra** [comment]
@awholegunch Please take another look.

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**